### ISSUE TITLE
Rule S1313 should not raise when loopback, broadcast addresses are used

### ISSUE THREAD

**Issue by eric-therond-sonarsource** [SonarSource/sonar-dotnet#3450]
The subtask C#/VB RSPEC-1313 inherits now the complete description from [the main RSPEC](https://jira.sonarsource.com/browse/RSPEC-1313) and in particular this text block about exceptions:
```
Loopback addresses 127.0.0.0/8 in CIDR notation (from 127.0.0.0 to 127.255.255.255)
Broadcast address 255.255.255.255
Non routable address 0.0.0.0
Strings of the form 2.5.<number>.<number> as they often match Object Identifiers (OID).
```

So, the rule should be updated to not raise when loopback, broadcast, non routable addresses are used and when OIDs are used.
